commit 2ba3e031847702049ecd99138a4ae7c2040ea5be (refs/changes/22/3622/3)
Author: Jan Wyszynski <jan.wyszynski@joyent.com>
Date:   2018-03-20T01:19:21+00:00 (1 year, 7 months ago)
    
    joyent/node-zkstream#32 ZKBuffer emits unhandled error on packet opcode CLOSE_SESSION
    Reviewed by: Alex Wilson <alex.wilson@joyent.com>
    Approved by: Alex Wilson <alex.wilson@joyent.com>

diff --git a/lib/zk-buffer.js b/lib/zk-buffer.js
index 51549f4..3556118 100644
--- a/lib/zk-buffer.js
+++ b/lib/zk-buffer.js
@@ -317,6 +317,7 @@ ZKBuffer.prototype.readResponse = function (xidMap) {
 		case 'PING':
 		case 'SYNC':
 		case 'DELETE':
+		case 'CLOSE_SESSION':
 			/*
 			 * No special response packet format -- error code in
 			 * header determines status of the request.
